app/src/main/java/com/nibalk/tasky/navigation/NavigationRoot.kt

```diff
@@ -0,0 +1,51 @@
+package com.nibalk.tasky.navigation
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import com.nibalk.tasky.auth.presentation.login.LoginScreenRoot
+import com.nibalk.tasky.auth.presentation.register.RegisterScreenRoot
+
+@Composable
+fun NavigationRoot(
+    navController: NavHostController,
+) {
+    NavHost(
+        navController = navController,
+        startDestination = Route.AUTH_NAV_GRAPH
+    ) {
+        authGraph(navController)
+    }
+}
+
+private fun NavGraphBuilder.authGraph(navController: NavHostController) {
+    navigation(
+        startDestination = Route.AUTH_LOGIN_SCREEN,
+        route = Route.AUTH_NAV_GRAPH
+    ) {
+        composable(route = Route.AUTH_LOGIN_SCREEN) {
```

Comment:
Did you know that you don't need routes anymore? 😄 
https://youtu.be/AIC_OFQ1r3k?si=X5ay1-8OZXGvh1UC

Reply:
Wow this is cool. 🎉 
It's bad that I missed this video. I will fix that in an upcoming PR :)